assets/js/theme/global/featuredProductsQuery.js

```diff
@@ -64,7 +64,7 @@ function firstProductOutput(product, featuredDescription) {
                 <div class="card-summary">
                     ${featuredDescription}
 
-                    <a href="${product.node.path}">Learn More »</a>
+                   
```

Comment:
Not a big deal, but going forward, lets get rid of the empty space between sections that aren't distinct and limit it to a single line when it is distinct. I haven't been great about this myself, we'll need to go back in and refactor at some point